.github/workflows/lol-hackathoon.yml

```
@@ -0,0 +1,310 @@
+jobs:
+  # ---------------------------------------------------------------------------
+  #  1) Copy the block below (the "restart-deployment" job), 
+  #  2) Rename each job uniquely (job1, job2, job3, ... job100),
+  #  3) Paste for as many times as you want to run it (100 times shown here).
+  # ---------------------------------------------------------------------------
+
+  job1:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)